<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PnlVotacao</class>
 <widget class="QMainWindow" name="PnlVotacao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Curso Senac DS- Tela Votacao Urna UC6 (14 11 2025)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="LblNumeroCandidatos">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LblTituloVotacao">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Votação</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblVoto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Voto:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TxtVoto">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnVotar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Votar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
